### PR TITLE
Updated usage example to add onAccessToken

### PR DESCRIPTION
**Pull request by rpeyron**
Hello,

Thanks for your useful component. Here is a very small PR to update usage example to add onAccessToken  and have a behavior more like one could expect just after have logged in without the need of refreshing the page (it took me some time to understand why, so it may help other users).

Hth,
Remi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ import { SpotifyAuth, Scopes } from 'react-spotify-auth'
 import 'react-spotify-auth/dist/index.css'
 
 const App = () => {
-  const token = Cookies.get('spotifyAuthToken')
+  let [token, setToken] = useState('');
```
Comment (kevin51jiang) [inline README.md]:
```suggestion
    const [token, setToken] = React.useState(Cookies.get("spotifyAuthToken"))
```

Just wanna keep it copy-pastable with vanilla React + ability to get token from storage

File: README.md [hunk 2]
```diff
@@ -63,6 +63,7 @@ const App = () => {
           redirectUri='http://localhost:3000/callback'
           clientID='1a70ba777fec4ffd9633c0c418bdcf39'
           scopes={[Scopes.userReadPrivate, 'user-read-email']} // either style will work
+          onAccessToken={(token:string) => { setToken(token)} }
```
Comment (kevin51jiang) [inline README.md]:
```suggestion
          onAccessToken={(token) => setToken(token)}
```

No type annotation to keep it good with vanilla JS

**Review by kevin51jiang — CHANGES_REQUESTED** [review]
Hey, thanks for the PR!

There's just a few things I would like to change, since the example should be compatible with vanilla JS React, and it should still be able to store the auth token in a cookie in case the user decides to close + reopen the browser.

**Comment by rpeyron** [comment]
Hi, I have included the improvements you suggested. Thanks

**Comment by kevin51jiang** [comment]
Thanks again :D

**Pull request merged**